src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SORTORDER;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.comparator.PersonComparator;
+
+/**
+ * Sorts the current filtered list based on the specified sort type.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the current list based on the specified field.\n"
```

Comment:
Would it be better to write that the fields can be `github`, `name` or `telegram` and that `order` can be `asc` or `desc`?

Reply:
Yup I agree with this. A brief message in the `MESSAGE_USAGE` would make it clearer for the reader and the rest can be in the User Guide.

Reply:
Right. I will add that in and change the comparsion as well.